policies/stable-release-updates.md

```diff
@@ -94,3 +94,6 @@ Patch release commits are obviously allowed (updates to CHANGES.md, NEWS.md,
 version, and copyright updates).
 
 **Exceptions to this policy require OTC approval.**
+
+*The policy for changes allowed in stable releases applies also to
+pre-release branches.*
```

Comment:
I don't think this is what we agreed. We said it applied to *beta* releases, but not alpha releases.

Reply:
We currently branch at beta, but yeah I can amend that.